examples/bpaf-app/Cargo.toml

```diff
@@ -9,4 +9,4 @@ name = "bpaf-app"
 path = "app.rs"
 
 [dependencies]
-bpaf = "0.4.7"
+bpaf = { version = "0.5.3", default-features = false }
```

Comment:
The default case is most likely what people will do so that is probably what we should demonstrate.

If there is a big enough difference to have a "min" variant, like clap, we can consider that.

Reply:
I don't think there will be any measurable difference, it's just something I tend to do automatically. Will push a fix.